aws/table_aws_cognito_user_group.go

```diff
@@ -0,0 +1,241 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cognitoidentityprovider"
+	"github.com/aws/aws-sdk-go-v2/service/cognitoidentityprovider/types"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCognitoUserGroup(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_cognito_user_group",
+		Description: "AWS Cognito User Group",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"group_name", "user_pool_id"}),
+			Hydrate:    getCognitoUserGroup,
+			Tags:       map[string]string{"service": "cognito-idp", "action": "GetGroup"},
```

Comment:
In https://github.com/turbot/steampipe-plugin-aws/blob/main/aws/table_aws_cognito_identity_pool.go#L24 we use the service tag as `cognito-identity`. Is the service tag correct?

Reply:
The AWS CLI command looks like `aws cognito-idp list-groups --user-pool-id <your_user_pool_id>`
https://docs.aws.amazon.com/cli/latest/reference/cognito-idp/list-groups.html